account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_mapping.py

```diff
@@ -12,6 +12,13 @@ class AccountBankStatementImportSheetMapping(models.Model):
     name = fields.Char(
         required=True,
     )
+
+    delete_rows = fields.Integer(
```

Comment:
actually, I'm seeing now it is only a naming issue...
![image](https://user-images.githubusercontent.com/23449160/121524948-1d904a80-c9f8-11eb-94be-0e883d5256e7.png)

Can you change this field no somthing more accurate: `skipped_rows` or similar

Reply:
Done! Thank you @alexey-pelykh and @LoisRForgeFlow.